C++/include/data_structure/stack/stack.hpp

```diff
@@ -0,0 +1,82 @@
+/*
+    Stack
+    ------------------
+    A data structure which has the Last In First Out (LIFO) property. Elements are added
+    and removed from the top(head). Only the top element can be accessed at any given time.
```

Comment:
Sorry, missed this before. Add a space between `top` and `(head)`.

Reply:
Thanks. This was my first open-source contribution and I had fun contributing. Let me know if there is anything else that needs to be changed.

Reply:
Great! Glad to hear you enjoyed it. Please don't hesitate to add more to the repo. 🙂